lib/mgmt_be_client.h

```diff
@@ -37,6 +37,15 @@ enum mgmt_be_client_id {
 	for ((id) = MGMTD_BE_CLIENT_ID_MIN;		\
 	     (id) < MGMTD_BE_CLIENT_ID_MAX; (id)++)
 
+#define FOREACH_BE_CLIENT_BITS(id, bits)                                       \
```

Comment:
Maybe something like:
```
static inline int get_next_be_client(int id, int bits)
{
        for (; id < MGMTD_BE_CLIENT_ID_MAX && IS_IDBIT_UNSET(bits, id); id++);
        return id;
}

#define FOREACH_BE_CLIENT_BITS (id, bits) \
        for ((id) = get_next_be_client(MGMTD_BE_CLIENT_ID_MIN, bits);
             (id) < MGMTD_BE_CLIENT_ID_MAX;
             (id) = get_next_be_client(id, bits))
```
To make checkpatch happy.

Reply:
BTW, all this "Client IDs" part of `lib/mbmt_be_client.h` should be moved to `mgmt_be_adapter.h`. It is not supposed to be used by any daemon other than mgmtd.

Reply:
yes, and the ID to name mapping array in mgmt_be_client.c.. which is going to out-date our example commits of how to add a daemon, sigh.

Reply:
We can make this a separate change, no problem.